From: "Matteo F. Vescovi" <mfv.debian@gmail.com>
Date: Fri, 21 Dec 2012 16:34:56 +0100
Subject: Fix_multiarch_paths

This second patch adds multiarch paths to the
standard ones for ilm and openexr libraries.
---
 src/cmake/modules/FindIlmBase.cmake | 2 ++
 src/cmake/modules/FindOpenEXR.cmake | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/src/cmake/modules/FindIlmBase.cmake b/src/cmake/modules/FindIlmBase.cmake
index 9e2fe5b..041b470 100644
--- a/src/cmake/modules/FindIlmBase.cmake
+++ b/src/cmake/modules/FindIlmBase.cmake
@@ -118,7 +118,9 @@ set (IlmBase_generic_include_paths
   /opt/local/include)
 set (IlmBase_generic_library_paths
   /usr/lib
+  /usr/lib/${CMAKE_LIBRARY_ARCHITECTURE}
   /usr/local/lib
+  /usr/local/lib/${CMAKE_LIBRARY_ARCHITECTURE}
   /sw/lib
   /opt/local/lib)
 
diff --git a/src/cmake/modules/FindOpenEXR.cmake b/src/cmake/modules/FindOpenEXR.cmake
index bd37e8a..955c025 100644
--- a/src/cmake/modules/FindOpenEXR.cmake
+++ b/src/cmake/modules/FindOpenEXR.cmake
@@ -114,7 +114,9 @@ set (OpenEXR_generic_include_paths
   /opt/local/include)
 set (OpenEXR_generic_library_paths
   /usr/lib
+  /usr/lib/${CMAKE_LIBRARY_ARCHITECTURE}
   /usr/local/lib
+  /usr/local/lib/${CMAKE_LIBRARY_ARCHITECTURE}
   /sw/lib
   /opt/local/lib)
 
